<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dConfigurarTerminal</class>
 <widget class="QDialog" name="dConfigurarTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registro de Usuarios</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Ticket/Version 1/img/logotipo1.png</normaloff>../Ticket/Version 1/img/logotipo1.png</iconset>
  </property>
  <widget class="QLineEdit" name="txtNombrePlaza">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLocalidadEstado">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLocalidadMunicipio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Atributo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Nombre de la Plaza&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;&lt;br/&gt;Localidad (Estado)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;&lt;br/&gt;Localidad Municipio&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;&lt;br/&gt;No.Terminal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Aceptar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid #5b8a3c;
   border-top: 1px solid #5b8a3c;
   background: #5b8a3c;
   padding: 9px 18px;
   border-radius: 9px;
   color: white;
	font: 75 16pt &quot;Roboto Condensed&quot;;
   /*text-decoration: none;*/
   vertical-align: middle;

}
  
QPushButton:hover {
   border-top-color:rgb(77, 207, 57);
	background: rgb(77, 207, 57);
   }</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Titulo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Return To Sender</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Configurar Terminal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancelar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   border:1px solid rgb(165, 0, 0);
   border-top: rgb(165, 0, 0);
   background: rgb(165, 0, 0);
   padding: 9px 18px;
   border-radius: 9px;
   color: white;
   font-size: 16px;
	font: 75 16pt &quot;Roboto Condensed&quot;;
   /*text-decoration: none;*/
   vertical-align: middle;

}
  
QPushButton:hover {
   border-top-color: rgb(235, 0, 3);
	background:rgb(235, 0, 3);
   }</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Imagen">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>461</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Ticket/Version 1/img/logotipo1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
